### PR TITLE
fix(widget-currency-rates): apply currency rates

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

[QA - 319 - 1](https://pyengine.atlassian.net/browse/QA-319)

- Add `display/loadCurrencyRates` dispatch in `DashboardsContainer`
- Apply currency rates in `MonthlyCostWidget` and `BudgetUsageSummaryWidget`

image

### CODE REVIEW

File: src/services/dashboards/DashboardsContainer.vue [hunk 1]
```diff
@@ -30,6 +32,12 @@ export default {
     components: { DashboardsLNB, GeneralPageLayout, VerticalPageLayout },
     setup() {
         const { breadcrumbs } = useBreadcrumbs();
+
+        /* init */
+        (async () => {
+            await store.dispatch('display/loadCurrencyRates');
+        })();
```
Comment (yuda110) [inline src/services/dashboards/DashboardsContainer.vue:39]: suggestion: How about get currencyRates in `DashboardWidgetContainer.vue`?

**Review by yuda110 — APPROVED** [review]
LGTM!!! I left a comment!

**Review by song2kim — APPROVED** [review]
LGTM~ Thank you

File: src/services/dashboards/DashboardsContainer.vue (hunk 1 above)
Comment (piggggggggy) [reply src/services/dashboards/DashboardsContainer.vue:39]: Ah.. that seems more efficient.

**Comment by piggggggggy** [comment]
Moved currency rates init function to `DashboardWidgetContainer.vue`.

**Pull request merged**